### ISSUE TITLE
Code sample is calculating offsetX/Y incorrectly

### ISSUE THREAD

**Issue by ChumpChief** [WICG/ViewportAPI#17]
Currently:

`var offsetX = viewport.scrollLeft;`
`var offsetY = viewport.clientHeight - document.documentElement.clientHeight + viewport.scrollTop;`

Should instead be this I think:

`var offsetX = viewport.scrollLeft - document.scrollingElement.scrollLeft;`
`var offsetY = document.documentElement.clientHeight - (viewport.scrollTop - document.scrollingElement.scrollTop) - viewport.clientHeight;`

**Comment by bokand** [comment]
The bar is position: fixed so it's already fixed to the layout viewport, no need to compensate by scrollingElement.scrollLeft/scrollTop.

I've tried this out on Chrome OSX and it works. I just tried on Chrome Android and it's broken because documentElement isn't the scrollingElement. Because of some experiments with how we handle the URL bar I had to more significantly change how we get the layout viewport height but I've updated the sample and it should work now.
